smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/endpointsV2/RuleSetParameterFinder.java

```diff
@@ -90,6 +106,11 @@ public Map<String, String> getStaticContextParamValues(OperationShape operation)
                     value = "`" + definition.getValue().expectStringNode().toString() + "`";
                 } else if (definition.getValue().isBooleanNode()) {
                     value = definition.getValue().expectBooleanNode().toString();
+                } else if (definition.getValue().isArrayNode()) {
+                    ArrayNode arrayNode = definition.getValue().expectArrayNode();
+                    value = arrayNode.getElements().stream()
+                      .map(element -> "'" + element.expectStringNode().getValue() + "'")
```

Comment:
Probably need string escape detection here. Also recommend using triple quotes in Java to enable double quotes in JS.